robot/impl/resource_manager_test.go

```diff
@@ -462,13 +461,6 @@ func TestManagerAdd(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, resource1, test.ShouldEqual, injectBoard)
 
-	injectMotionService := &inject.MotionService{}
```

Comment:
This part of the test does not seem to add anything and means that anyone importing resource also needs to import nlopt by transitive dependencies (although the reason resource is importing this is unclear to me)